www-client/qutebrowser/qutebrowser-9999.ebuild

```diff
@@ -31,6 +32,10 @@ RDEPEND="${COMMON_DEPEND}
 	>=dev-python/pygments-2.1.3[${PYTHON_USEDEP}]
 	>=dev-python/pypeg2-2.15.2[${PYTHON_USEDEP}]
 	dev-python/PyQt5[${PYTHON_USEDEP},declarative,gui,network,printsupport,webkit,widgets]
```

Comment:
Webengine can be enabled at execution time with `--backend webengine`. The two may coexist.